scenarioo-client/app/components/detailarea/detailarea.component.html

```diff
@@ -15,23 +15,32 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic" id="sc-showHideDetailsButton"
-        class="btn btn-primary pull-right" type="button">
-    {{ isPanelCollapsed ? "Show details" : "Hide details" }}
-</button>
+<div class="sc-meta-data-panel row">
 
-<div #collapse="bs-collapse" [collapse]="isPanelCollapsed">
-    <div id="sc-metadata-panel" class="detail-container col-sm-4 pull-right">
-        <div class="detail-row">
+    <button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic"
+            class="btn btn-primary pull-right" id="sc-showHideDetailsButton" type="button">
+        {{ isPanelCollapsed ? "Show details" : "Hide details" }}
+    </button>
+
+    <div [ngClass]="isPanelCollapsed ? 'col-sm-11' : 'col-sm-8'">
+        <ng-content></ng-content>
+    </div>
```

Comment:
just awesome - look how cool that the detailsarea finally fully controls how to do the outer layout depending on collapsed state :-) 👍